enterprise/rules.py

```diff
@@ -22,26 +22,35 @@ def has_implicit_access_to_dashboard(user, obj):  # pylint: disable=unused-argum
     """
```

Comment:
nit: is the `# pylint: disable=unused-argument` comment relevant here now that we're using `obj`? Similar to the other functions, as well.

Reply:
haha, actually, `user` is still unused.  It does apply to the `...explicit_access...` one, though.